app/src/services/api.ts

```diff
@@ -26,5 +29,28 @@ export const api = createApi({
       transformResponse: (response: { data: InventoryProgressResponse }) =>
         response.data,
     }),
+    addCity: builder.mutation<CityAttributes, {}>({
+      query: (data) => ({
+        url: `/city`,
+        method: "POST",
+        body: data,
+      }),
+    }),
+  }),
+});
+
+export const openclimateAPI = createApi({
+  reducerPath: "openclimateapi",
+  baseQuery: fetchBaseQuery({
+    baseUrl: "https://openclimate.openearth.dev",
```

Comment:
This URL should probably depend on the environment we're using, for prod we should use the prod OpenClimate API. You can extract it to an env var or decide which one to use based on `process.env.NODE_ENV === 'production'`